catapult/config.py

```diff
@@ -12,7 +12,9 @@
     sys.exit(1)
 
 # git config
-GIT_REPO = os.environ.get("CATAPULT_GIT_REPO", "./")
+GIT_REPO = os.environ.get("CATAPULT_TARGET_GIT_REPO") or os.environ.get(
+    "CATAPULT_GIT_REPO", "./"
+)
```

Comment:
Out of interest why did you decide to change the name of this env var?

Reply:
We had an explanation in the README calling out that this is env var should be the _target_ repo, not the catapult repo. I figured changing the name would help clear up the confusion.

Reply:
Ah ok, that's a good enough reason. Thanks!